test/extern/diten/externfaston.chpl

```diff
@@ -0,0 +1,27 @@
+use CommDiagnostics;
+require "externfaston.c", "externfaston.h";
+
+pragma "fast-on safe extern function"
+extern proc foo1(): int;
+extern proc foo2(): int;
+
+pragma "fast-on safe extern function"
+proc nonExternFunction() {
```

Comment:
I was initially thinking about adding an error message when the pragma is used somewhere other than extern functions, but decided to silently ignore it instead.  I forgot to remove that declaration from my test and only noticed it this morning.  I'll remove it.